contactaccessrights/js/src/access-rights/controllers/access-rights-ctrl.js

```diff
@@ -0,0 +1,22 @@
+define(['access-rights/modules/controllers'], function (controllers) {
```

Comment:
Still for consistency, define the modules like this (even if it's just with one dependency)
```js
define([
    'dependency 1',
    'dependency 2',
    // ...
    'dependency 3'
], function (1, 2, 3) {

});
```